views/login.jade

```diff
@@ -21,9 +21,9 @@ block content
                     if message
                         div(style= "text-align:center" class="alert alert-"+ type +"" role="alert")= message
                         //alert-success
-                        //alert alert-info"
-                        //alert alert-warning
-                        //alert alert-danger
+                        //alert-info"
+                        //alert-warning
+                        //alert-danger
```

Comment:
Maybe we should have some documentation here saying what this means. Perhaps make this say that these are the options for the flash screen